test/bats/helpers/lxly-bridge.bash

```diff
@@ -356,3 +357,60 @@ function wait_for_expected_token() {
         sleep "$poll_frequency"
     done
 }
+
+function get_claims() {
+    local aggkit_node_url=$1
+    local l1_rpc_network_id=$2
+    local expected_global_index=$3
+    local max_attempts=$4
+    local poll_frequency=$5
+    local attempt=0
+
+    while true; do
+        ((attempt++))
+        echo "🔍 Attempt $attempt: fetching claims from the RPC..." >&3
```

Comment:
FYI it is better to use the `log` function, because it includes timestamp as well (also in that case `>&3` redirection should be removed)

```suggestion
        log "🔍 Attempt $attempt: fetching claims from the RPC ("$aggkit_node_url")..."
```